dart/lib/src/sentry_client.dart

```diff
@@ -221,6 +227,41 @@ class SentryClient {
     return captureEvent(event, scope: scope, hint: hint);
   }
 
+  @internal
+  Future<SentryId> captureTransaction(
+    SentryTransaction transaction, {
+    Scope? scope,
+  }) async {
+    SentryTransaction? preparedTransaction =
+        _prepareEvent(transaction) as SentryTransaction;
+
+    if (scope != null) {
+      preparedTransaction =
+          await scope.applyToEvent(preparedTransaction) as SentryTransaction?;
+    } else {
+      _options.logger(SentryLevel.debug, 'No scope is defined');
```

Comment:
l: 
What about something more descriptive:
```suggestion
      _options.logger(SentryLevel.debug, 'No scope to apply on transaction was provided');
```